### PR TITLE
[NET 6 Attribute Conversion] Autotoolbox

### PR DESCRIPTION
**Pull request by chamons**

### CODE REVIEW

**Review by rolfbjarne — APPROVED** [review]

**Review by mandel-macaque — APPROVED** [review]

File: src/AudioToolbox/AudioSession.cs [hunk 1]
```diff
@@ -168,10 +171,13 @@ NSArray Extract (IntPtr key, NSString secondKey)
 			}
 		}
 
-#if !NET
-		[Deprecated (PlatformName.iOS, 7, 0)]
-#else
+#if NET
 		[UnsupportedOSPlatform ("ios7.0")]
+#if IOS
+		[Obsolete ("Starting with ios7.0.", DiagnosticId = "BI1234", UrlFormat = "https://github.com/xamarin/xamarin-macios/wiki/Obsolete")]
```
Comment (mandel-macaque) [inline src/AudioToolbox/AudioSession.cs:177]: @chamons  we are using ""Starting with macos10.14 not available anymore."" I think you have an issue with that message, am I wrong?

**Review by mandel-macaque — CHANGES_REQUESTED** [review]

**Review by dalexsoto — APPROVED** [review]

File: src/AudioToolbox/AudioSession.cs (hunk 1 above)
Comment (dalexsoto) [reply src/AudioToolbox/AudioSession.cs:177]: I think this is fine, I mean it is `obsolete` - `starting with iOS7.0.` also if people need more context we already include the `UrlFormat` pointing to further explanation

Comment (dalexsoto) [reply src/AudioToolbox/AudioSession.cs:177]: On the other hand I just realized what Manuel means so yeah for consistency with the other PRs -> `Starting with ios7.0 not available anymore.`

Comment (dalexsoto) [reply src/AudioToolbox/AudioSession.cs:177]: I think the confusion is coming from this one https://github.com/xamarin/xamarin-macios/pull/13791#discussion_r788221530 so here the tool is doing the right thing and nothing needs to be changed.

File: src/AudioToolbox/AudioFile.cs [hunk 2]
```diff
@@ -53,11 +53,15 @@ public enum AudioFileType {  // UInt32 AudioFileTypeID
 		AIFF = 0x41494646, // AIFF
 		AIFC = 0x41494643, // AIFC
 		WAVE = 0x57415645, // WAVE
-#if !NET
-		[NoWatch, iOS (11,0), Mac(10,13), TV (11,0)]
-#else
+#if NET
 		[SupportedOSPlatform ("ios11.0")]
+		[SupportedOSPlatform ("macos10.13")]
```
Comment (tj-devel709) [inline src/AudioToolbox/AudioFile.cs:58]: Flagging just in case since lowest supported is 10.14

File: src/AudioToolbox/AudioSession.cs (hunk 1 above)
Comment (chamons) [reply src/AudioToolbox/AudioSession.cs:177]: The error you get from dotnet build looks like:

/Users/donblas/tmp/obsolete/Program.cs(4,1): warning BI1234: 'Foo.Bar()' is obsolete: 'Starting with ios7.0.' [/Users/donblas/tmp/obsolete/obsolete.csproj]

So I'm pretty happy with that. Thoughts @mandel-macaque ?

**Review by mandel-macaque — APPROVED** [review]

**Pull request merged**